test/unit/middlewares/extensionRequests.test.ts

```diff
@@ -0,0 +1,71 @@
+import chai from "chai";
+import sinon from "sinon";
+const { expect } = chai;
+
+import { createExtensionRequestValidator } from "../../../middlewares/validators/extensionRequestsv2";
+import { extensionCreateObject } from "../../fixtures/extension-requests/extensionRequests";
+import { REQUEST_STATE } from "../../../constants/requests";
+
+describe("Extension Request Validators", function () {
```

Comment:
I Have this as a question in my mind, as I don't know much about testing. why is it working fine??

What I mean is you have used the request type "ExtensionRequestRequest" in side the function, but while testing we are using req type any. shouldn't it trow an error for expecting req type "ExtensionRequestRequest" but got any.